src/main/kotlin/org/pkl/lsp/documentation/toMarkdown.kt

```diff
@@ -229,6 +229,34 @@ private fun showDocCommentAndModule(node: PklNode?, text: String, context: PklPr
       appendLine()
       appendLine()
       append("in [${module.moduleName}](${module.getLocationUri(forDocs = true)})")
+
+      val docUrl = node?.getDocumentationUrl()
+      if (docUrl != null) {
+        val domain =
+          try {
+            java.net.URI(docUrl).host
+          } catch (e: Exception) {
+            "documentation"
```

Comment:
This would be a misconfiguration error if this happens; seems like we should catch this at configuration time.